src/citrine/resources/project.py

```diff
@@ -157,6 +157,13 @@ def share(self,
             "resource": {"type": resource_type, "id": resource_id}
         })
 
+    def global_share(self,
```

Comment:
How about `make_public`

Reply:
Or toggle public

Reply:
(to weigh in, I prefer make_public and make_private, toggle is too implicit for an access control feature IMO)